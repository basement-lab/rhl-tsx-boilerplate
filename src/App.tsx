import * as React from "react";

export default class App extends React.Component<any,any> {
    render() {
        return (<div>
            <h1>React Hot Loader v3 + Typescript</h1>
            <div>
                Test hot loading here
            </div>
        </div>);
    }
}