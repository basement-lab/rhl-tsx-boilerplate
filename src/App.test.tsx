
import * as React from 'react';
import { mount } from 'enzyme';

import App from './App';

/** ************************************************************************* */

test('[App] mounts', () => {
  const wrapper = mount(<App />);

  expect(wrapper).toMatchSnapshot();
});
