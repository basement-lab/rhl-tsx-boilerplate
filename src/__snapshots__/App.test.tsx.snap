// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[App] mounts 1`] = `
.c0 {
  color: red;
  height: 100px;
  width: 200px;
}

<App>
  <div>
    <h1>
      React Hot Loader v3 + Typescript
    </h1>
    <Styled(Button)>
      <Button
        className="c0"
      >
        <button
          onClick={[Function]}
        >
          Click Me!
        </button>
      </Button>
    </Styled(Button)>
    <div>
      Test hot loading here
    </div>
  </div>
</App>
`;
